src/UiElement/UiElementFormOptionsTrait.php

```diff
@@ -25,6 +25,7 @@ trait UiElementFormOptionsTrait
     public function getFormOptions(): array
     {
         try {
+            /** @phpstan-ignore-next-line */
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** You can ignore this error like this when it's one line return : 
`/** @var array*/`
```suggestion
            /** @var array*/
```

‏